docs/GenericsManifesto.md

```diff
@@ -272,6 +283,8 @@ Unlike the minor extensions, major extensions to the generics model provide more
 
 ### Conditional conformances (*)
 
+*This feature has been accepted in [SE-0143](https://github.com/apple/swift-evolution/blob/master/proposals/0143-conditional-conformances.md) and is planned for release in SWIFT_VERSION_HERE.*
```

Comment:
Can someone from the core team confirm the version of Swift this will be released in? 3.1 or 4?